src/core/quicdef.h

```diff
@@ -105,7 +105,7 @@ typedef struct QUIC_PATH QUIC_PATH;
 //
 // Maximum number of partitions to support.
 //
-#define QUIC_MAX_PARTITION_COUNT                64
+#define QUIC_MAX_PARTITION_COUNT                128
```

Comment:
Any specific reasons now to limit this to 128? EPYC is now at 256 threads in a dual socket system, and over the next few years a setup like that will just get cheaper and more common.

Reply:
We currently use a `uint8_t` for partition count, so `256` doesn't fix. We can change it to a `uint16_t` in a future change.